src/syscall/exec_unix.go

```diff
@@ -0,0 +1,7 @@
+//go:build unix
+
+package syscall
+
+// Implemented in runtime package.
+func runtime_BeforeExec()
+func runtime_AfterExec()
```

Comment:
Looks like these are never called? Why did you declare them?

Reply:
These were leftover stubs back when https://github.com/tinygo-org/tinygo/pull/4426 was not merged yet.